pyproject.toml

```diff
@@ -165,6 +165,8 @@ connectorx="0.3.1"
 dbt-core=">=1.2.0"
 dbt-duckdb=">=1.2.0"
 pymongo = ">=4.3.3"
+qdrant-client=">=1.6.9"
```

Comment:
please remove those dependencies. best if you could revert to the original files. then use poetry -E to install qdrant in yml workflow file. see how we do it with `weaviate`

two things to note
1. this is how extra is defined:
`qdrant-client = {version = "^1.6.4", optional = true, extras = ["fastembed"]}`
so actually qdrant will pick fastembed itself

2. and I agree that qdrant should fix this thing. maybe we'll ask them on meetup tomorrow.